components/dialog/test/dialog-ifrau-contents.vdiff.html

```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
```

Comment:
This was the last remaining usage of this, so moved it right in here. This is a very specific case, needing to load an html file - all our other vdiff tests have this handled by the library automatically.